cpp/ps0000/solution 1.cpp

```diff
@@ -1,7 +1,9 @@
-#include<iostream>
+#include <iostream> // 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`